blocks/columns/columns.css

```diff
@@ -7,27 +7,122 @@
   width: 100%;
 }
 
-.columns > div > div {
+.columns>div>div {
   order: 1;
 }
 
-.columns > div > .columns-img-col {
+.columns>div>.columns-img-col {
   order: 0;
 }
 
-.columns > div > .columns-img-col img {
+.columns>div>.columns-img-col img {
   display: block;
 }
 
+.dotw {
```

Comment:
The best practice is to make your selectors specific to this block.
Continue to use `.columns` on each new selector that you create.

Reply:
when .columns is added .dotw no longer works, because both class are giving to the parent element.
please advise on how to approach this to keep with best practice.

Reply:
I relized what the issue was while working on the carousel component and fixed the selectors for the columns-block